### ISSUE TITLE
Interface for filtering "sources"

### ISSUE THREAD

**Issue by hombit** [lincc-frameworks/pandas-ts#7]
The interface should make it simple and safe way to filter underlying source data

**Comment by hombit** [comment]
The current approach is doing something like:
```python
import pandas as pd
from pandas_ts.packer import pack_flat

sources = pd.DataFrame({id=..., time=..., mag=...})
packed: pd.Series = pack_flat(sources, name="sources")
# Do something with packed
new_sources = packed.ts.to_flat()
new_sources = new_sources[new_sources['mag'] < 99]
new_packed = pack_flat(new_sources, name="new_sources")
```

**Comment by hombit** [comment]
Could be also solved with #9

**Comment by hombit** [comment]
Implemented with #34 and #32
